scipy/conftest.py

```diff
@@ -153,6 +155,34 @@ def check_fpu_mode(request):
 )
 
 
+# Following the approach of NumPy's conftest.py...
```

Comment:
Now that the `hypothesis` PR is merged, this PR can use a rebase and then the changes here will disappear from the diff.

Reply:
Done.